cns/kubecontroller/nodenetworkconfig/conversion.go

```diff
@@ -87,14 +87,23 @@ func CreateNCRequestFromStaticNC(nc v1alpha.NetworkContainer) (*cns.CreateNetwor
 	if err != nil {
 		return nil, errors.Wrapf(err, "invalid SubnetAddressSpace %s", nc.SubnetAddressSpace)
 	}
-	subnet := cns.IPSubnet{
-		IPAddress:    primaryPrefix.Addr().String(),
-		PrefixLength: uint8(subnetPrefix.Bits()),
+
+	var subnet cns.IPSubnet
+	if nc.Type == v1alpha.VNETBlock {
+		subnet.IPAddress = nc.NodeIP
+		subnet.PrefixLength = uint8(subnetPrefix.Bits())
```

Comment:
nit: looks like subnet.PrefixLength is common for both cases

Reply:
Fixed